browser/src/control/jsdialog/Definitions.Menu.ts

```diff
@@ -180,524 +180,545 @@ enum functionCategories {
-	_('COUPDAYSNC'),
-	_('COUPNDC'),
-	_('COUPNUM'),
-	_('COUPPCD'),
-	_('CUMIPMT'),
-	_('CUMIPMT_ADD'),
-	_('CUMPRINC'),
-	_('CUMPRINC_ADD'),
-	_('DB'),
-	_('DDB'),
-	_('DISC'),
-	_('DOLLARDE'),
-	_('DOLLARFR'),
-	_('DURATION'),
-	_('EFFECT'),
-	_('EFFECT_ADD'),
-	_('FV'),
-	_('FVSCHEDULE'),
-	_('INTRATE'),
-	_('IPMT'),
-	_('IRR'),
-	_('ISPMT'),
-	_('MDURATION'),
-	_('MIRR'),
-	_('NOMINAL'),
-	_('NOMINAL_ADD'),
-	_('NPER'),
-	_('NPV'),
-	_('ODDFPRICE'),
-	_('ODDFYIELD'),
-	_('ODDLPRICE'),
-	_('ODDLYIELD'),
-	_('OPT_BARRIER'),
-	_('OPT_PROB_HIT'),
-	_('OPT_PROB_INMONEY'),
-	_('OPT_TOUCH'),
-	_('PDURATION'),
-	_('PMT'),
-	_('PPMT'),
-	_('PRICE'),
-	_('PRICEDISC'),
-	_('PRICEMAT'),
-	_('PV'),
-	_('RATE'),
-	_('RECEIVED'),
-	_('RRI'),
-	_('SLN'),
-	_('SYD'),
-	_('TBILLEQ'),
-	_('TBILLPRICE'),
-	_('TBILLYIELD'),
-	_('YDB'),
-	_('XIRR'),
-	_('XNPV'),
-	_('YIELD'),
-	_('YIELDDISC'),
-	_('YIELDMAT'),
+const financialFunctions: Array<FunctionNameAlias> = [
+	{ en: 'ACCRINT', de: 'AUFGELZINS' },
+	{ en: 'ACCRINTM', de: 'AUFGELZINSF' },
+	{ en: 'AMORDEGRC', de: 'AMORDEGRK' },
+	{ en: 'AMORLINC', de: 'AMORLINEARK' },
+	{ en: 'COUPDAYBS', de: 'ZINSTERMTAGVA' },
+	{ en: 'COUPDAYS', de: 'ZINSTERMTAGE' },
+	{ en: 'COUPDAYSNC', de: 'ZINSTERMTAGNZ' },
+	{ en: 'COUPNUM', de: 'ZINSTERMZAHL' },
+	{ en: 'COUPPCD', de: 'ZINSTERMVZ' },
+	{ en: 'CUMIPMT', de: 'KUMZINSZ' },
+	{ en: 'CUMIPMT_ADD', de: 'KUMZINSZ_ADD' },
+	{ en: 'CUMPRINC', de: 'KUMKAPITAL' },
+	{ en: 'CUMPRINC_ADD', de: 'KUMKAPITAL_ADD' },
+	{ en: 'DB', de: 'GDA2' },
+	{ en: 'DDB', de: 'GDA' },
+	{ en: 'DISC', de: 'DISAGIO' },
+	{ en: 'DOLLARDE', de: 'NOTIERUNGDEZ' },
+	{ en: 'DOLLARFR', de: 'NOTIERUNGBRU' },
+	{ en: 'DURATION', de: 'LAUFZEIT' },
+	{ en: 'EFFECT', de: 'EFFEKTIV' },
+	{ en: 'EFFECT_ADD', de: 'EFFEKTIV_ADD' },
+	{ en: 'FV', de: 'ZW' },
+	{ en: 'FV', de: 'Zielwert' },
+	{ en: 'FVSCHEDULE', de: 'ZW2' },
+	{ en: 'INTRATE', de: 'ZINSSATZ' },
+	{ en: 'IPMT', de: 'ZINSZ' },
+	{ en: 'IRR', de: 'IKV' },
+	{ en: 'ISPMT', de: 'ISPMT' },
+	{ en: 'MDURATION', de: 'MLAUFZEIT' },
+	{ en: 'MIRR', de: 'QIKV' },
+	{ en: 'NOMINAL', de: 'NOMINAL' },
+	{ en: 'NOMINAL_ADD', de: 'NOMINAL_ADD' },
+	{ en: 'NPER', de: 'ZZR' },
+	{ en: 'NPER', de: 'Zeitraum' },
+	{ en: 'NPV', de: 'NBW' },
+	{ en: 'ODDFPRICE', de: 'UNREGERKURS' },
+	{ en: 'ODDFYIELD', de: 'UNREGERREND' },
+	{ en: 'ODDLPRICE', de: 'UNREGLEKURS' },
+	{ en: 'ODDLYIELD', de: 'UNREGLEREND' },
+	{ en: 'OPT_BARRIER', de: 'OPT_BARRIER' },
+	{ en: 'OPT_PROB_HIT', de: 'OPT_PROB_HIT' },
+	{ en: 'OPT_PROB_INMONEY', de: 'OPT_PROB_INMONEY' },
+	{ en: 'OPT_TOUCH', de: 'OPT_TOUCH' },
+	{ en: 'OPT_TOUCH', de: 'Funktion OPT_TOUCH' },
+	{ en: 'OPT_TOUCH', de: 'OPT_TOUCH' },
+	{ en: 'PDURATION', de: 'PLAUFZEIT' },
+	{ en: 'PMT', de: 'RMZ' },
+	{ en: 'PMT', de: 'Zahlungen' },
+	{ en: 'PPMT', de: 'KAPZ' },
+	{ en: 'PRICE', de: 'KURS' },
+	{ en: 'PRICEDISC', de: 'KURSDISAGIO' },
+	{ en: 'PRICEMAT', de: 'KURSFÄLLIG' },
+	{ en: 'PV', de: 'BW' },
+	{ en: 'PV', de: 'Barwert' },
```

Comment:
Duplications like this in this mapping are wrong. I'm not sure what we will get. But probably the wrong translation in the end. You should only process translations\source\de\formula\messages.po